README.md

```diff
@@ -16,35 +16,30 @@
 
 ## Project Description
-The Bodacious Advisor for Cuisine Over N-joyment ranks the restaurants within London based on Twitter reviews.
 
-## The Bacon Approach
-Our first step is to use a dataset provided by Yelp to get reviews and their associated Star Evaluation with a parser to get this key-value pair. This new dataset is used to train algorithms that are used for sentiment analysis so that we can read tweets for restaurants and predict its Star Evaluation (*-tweet). Reviews from Yelp will only be used for training the Ranking Algorithm. 
-
-Using a list with 2380 restaurants based in London with Tweeter account, we use Twitter-API to get tweets that mention those accounts so that we can apply our sentiment analysis algorithm on them, discovering their *-tweet. Our time window is three months from now, so we will not consider old tweets.
-
-//toDo Machine Learning (Sara)
+Online reviews have risen as one of the major factors that influence purchase behaviours of today’s consumers. Thanks to sites like Yelp and, as well as social networks like Twitter. Those reviews provide strong word-of-mouth power that can draw or drive away potential customers.
 
-//toDo ranking algorithm (Sara)
+Companies now embrace reviews as part of their marketing strategy. A Search Engine Land report found as much as 83% of marketers believe managing reviews produces good ROI. Managed correctly, online reviews can also support and improve the effectiveness of your social media strategy.
 
-//toDo weight model (Sara)
+The Bodacious Advisor for Cuisine Over N-joyment ranks the restaurants within London based on their online reviews on Twitter. 
 
 ## Architecture Overview
-The B.A.C.O.N pipeline for ranking restaurants is built on top of 6 components: 
+The B.A.C.O.N pipeline for ranking restaurants is built on top of 5 components: 
 
-1. A client interface to access the ranked data;
-2. An API to enables the access to data inside the storage;
-3. A Data Translation Layer for cleaning and normalising data from Yelp Dataset and Twitter List Members;
-4. A Data Process Layer, responsible for getting tweets about restaurants and formatting the data into our standard before store it;
-5. The Data Analytics Layer stands on top of DataIku, where our team of Data Scientists perform sentiment analysis/ranking algorithms and weight models to get the Restaurant Rank;
-6. We also have a Storage system to keep our precious data protected.
+1. An API to enables the access the Ranked Data inside the storage through an HTTP request;
+2. A Data Transform Layer (DTL) for cleaning and normalising data from Yelp Dataset and Twitter List Members;
+3. A Data Process Layer (DPL), responsible for getting tweets about restaurants and formatting the data into our standard before store it;
+4. The Data Analysis Layer (DAL) stands on top of DataIku, where our team of Data Scientists perform sentiment analysis/ranking algorithms and weight models to get the Restaurant Rank;
```

Comment:
Please consider changing to "A Data Analysis Layer (DAL), which stands on top of DataIku, where sentiment analysis/ranking algorithms and weight models are applied by our team of Data Scientists to obtain the Restaurant Rank ;"